scripts/benchall.py

```
@@ -0,0 +1,132 @@
+        #if bencher == clp_presto_bench and dataset_name == 'mongod':
+        if bencher == clp_presto_bench:
+            kwargs["dataset_variation"] = "cleaned_log"
+
+        print(f'Benchmarking {bencher.__name__} ({kwargs}) on dataset {dataset_name}')
+
+        bench = bencher(bench_target, **kwargs)
+        bench.attach = attach
+        bench.run_applicable(dataset_name)
+        #bench.run_everything(['ingest', 'cold'])
+    except Exception as e:
+        statement = f"{datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')}: {bencher.__name__} with argument {kwargs} failed on dataset {dataset_name}: {type(e).__name__}: {str(e)}"
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)